components/monitoring/base/grafanadashboard_sandbox_usage.yaml

```diff
@@ -0,0 +1,648 @@
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDashboard
+metadata:
+  name: sandbox-usage-dashboard
```

Comment:
We won't need this, but if you want a test dashboard, maybe the ones that Sergeii recommended would work.  If that's too much trouble for one PR, we can just start with a dashboard-less Grafana and add more dashboards in another PR.